src/components/Comments.jsx

```diff
@@ -0,0 +1,39 @@
+import { useParams } from "react-router-dom";
```

Comment:
Is it intentional that you only want to show the comments on their own and not with the review?

Reply:
yes, it was intentional, can change it tho